var/spack/repos/builtin/packages/met/package.py

```diff
@@ -41,6 +44,7 @@ class Met(AutotoolsPackage):
 
     depends_on("gsl")
     depends_on("bufr")
+    depends_on("proj", when="@12")
```

Comment:
Presumably for next major versions too?
```suggestion
    depends_on("proj", when="@12:")
```

Reply:
Done